src/java.base/share/classes/java/util/TimeZone.java

```diff
@@ -130,7 +130,10 @@
  * use is deprecated</strong> because the same abbreviation is often used
  * for multiple time zones (for example, "CST" could be U.S. "Central Standard
  * Time" and "China Standard Time"), and the Java platform can then only
- * recognize one of them.
+ * recognize one of them. The full list of the deprecated IDs can be viewed
```

Comment:
"deprecated IDs and their mappings" may read better, as SHORT_IDS is a map

Reply:
Good point, clarified the PR and CSR